### PR TITLE
Update garbage-collection.md

### PR DESCRIPTION
**Pull request by scjane**

### CODE REVIEW

File: registry/garbage-collection.md [hunk 1]
```diff
@@ -72,15 +72,15 @@ collection. Layer `a` had one reference removed but will not be garbage
...
+After garbage collection has been run, manifest `A` and its blobs remain.
 
 ```
 A -----> a
     \--> b
 ```
 
 
-## How Garbage Collection works
+## How Garbage Collection works?
```
Comment (mstanleyjones) [inline registry/garbage-collection.md]: No `?` needed here

Comment (scjane) [inline registry/garbage-collection.md]: Is the question mark (?) unnecessary in heading "Why Garbage Collection?", too? 
Is the question mark (?) required only if the heading is a complete sentence, like "How does garbage collection work?"? 
A little confused.

Comment (mstanleyjones) [inline registry/garbage-collection.md]: "How garbage collection works" is a sentence fragment. I would not have used something like "Why garbage collection" either, I probably would have called it "About garbage collection". Feel free to change it.

Comment (scjane) [reply registry/garbage-collection.md]: I think "About garbage collection" is similar to heading "What is garbage collection?", which already exists on this page. So, can we change "Why garbage collection" to "Why garbage collection is necessary"?

**Comment by scjane** [comment]
I see the reorganized headings and content. Thanks.

**Pull request merged**